src/model/jwt.ts

```diff
@@ -1,7 +1,23 @@
+import { Field, ObjectType } from "type-graphql";
+
 export interface UserInfo {
   id: string | undefined;
   email: string | undefined;
   username: string | undefined;
   firstName: string | undefined;
   lastName: string | undefined;
 }
+
+@ObjectType()
+export class UserInfo implements UserInfo {
```

Comment:
Possibly add java-doc explaining this (especially that it's a JWT).

Reply:
fixed